src/pages/Mentor/ChatNameBar/index.js

```diff
@@ -0,0 +1,59 @@
+import React, { useEffect } from "react";
+import { Typography, useMediaQuery } from "@material-ui/core";
+import CircleIcon from "@mui/icons-material/Circle";
+import InfoOutlinedIcon from "@mui/icons-material/InfoOutlined";
+import ArrowBackIosNewIcon from "@mui/icons-material/ArrowBackIosNew";
+import useStyles from "./styles";
+
+export default function ChatNameBar({ onBack, rooms, selectedRoomId }) {
+  const desktop = useMediaQuery("(min-width: 1200px)");
+  const laptop = useMediaQuery("(min-width: 769px) and (max-width: 1199px)");
+  const mobile = useMediaQuery("(max-width: 768px)");
+
+  const [roomName, setRoomName] = React.useState("");
+  const classes = useStyles({ desktop, laptop, mobile });
+
+  React.useEffect(() => {
+    rooms.map((room) => {
+      if (room.roomId === selectedRoomId) {
+        setRoomName(room.name);
+      }
+    });
```

Comment:
here as well try using ?. syntax if it happens the name is null from backend then we might get crash issues

Reply:
Okay